### DISCUSSION TITLE
划词翻译 & 朗读

### DISCUSSION THREAD

**Discussion by sleeper1023** [immersive-translate/immersive-translate#416] [Ideas]
### 新功能 | New Feature

1. 鼠标放在单个单词的时候可以翻译这个单词的含义，或者选中区域内容（比如单句和短语）后右键翻译也可以
2. 可以对这个功能在设置中加入一个开关，控制这个功能
3. 对指定区域进行英语朗读，可以锻炼听力

### 为什么您想要添加这些功能 | Why you want to add these features

1.段落翻译很方便，但是在想边学边看的时候，对于不认识的单词还要单独翻译看看，而且也不知道这个单词的读法
2. 这个功能可能有些人不需要，如果可以的话，让人自主选择是否开启这个选项最好啦

**Comment by versun** [comment]
总结下哈，你想要的功能是
1 划词翻译
2 原语言朗读

**Reply by sleeper1023** [reply]
对的对的，朗读希望可以支持选中范围朗读，谢谢你们的工作^_^
